iothub/service/src/Registry/Models/ServiceStatistics.cs

```diff
@@ -10,10 +10,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class ServiceStatistics
     {
+        /// <summary>
+        /// This constructor is for deserialization and unit test mocking purposes.
+        /// </summary>
```

Comment:
How about unifying the summaries across these classes?
```suggestion
        /// <summary>
        /// Creates an instance of this class. Provided for unit testing purposes only.
        /// </summary>
```